requirements.txt

```diff
@@ -4,7 +4,7 @@
 #
 #    pip-compile --annotation-style=line requirements.in
 #
--e git+https://github.com/the-maldridge/python-vote-core.git@f0b01e7e24f80673c4c237ee9e6118e8986cf0bb#egg=python3_vote_core  # via -r requirements.in
```

Comment:
Hi @mahmoud, this is the only change that I am afraid that it can break the legacy version. I made this change because of -e parameter, docker was not able to install this in the container. With -e, docker and toolforge deployment both is working.


Reset, you can merge PR without worry.

Reply:
Ah, that's a good callout. But it's fine to remove this, especially in production. To be extra cautious, we could create a quick backup of the legacy venv on toolforge.

Reply:
Okay